Insights/Account/UEBAAnomaly.yaml

```diff
@@ -0,0 +1,88 @@
+  DefaultTimeRange:
+    BeforeRange: 12d
+    AfterRange: 2d
+  ReferenceTimeRange:
+    BeforeRange: 0d
+  SingleValuesQuery: {}
+  TableQuery:
+    ColumnsDefinitions:
+    - Header: Anomaly
+      OutputType: String
+      SupportDeepLink: false
+    - Header: First Time User Anomaly
+      OutputType: Number
+      SupportDeepLink: true
+    - Header: Uncommon Among Peers
+      OutputType: Number
+      SupportDeepLink: true
+    - Header: Uncommon in Tenant
+      OutputType: Number
+      SupportDeepLink: true
+    - Header: First Time Tenant Anomaly
+      OutputType: Number
+      SupportDeepLink: true
+    QueriesDefinitions:
+    - Filter: " where (ActionType in('Add user','Update user','Reset user password') and  (ActivityInsights.FirstTimeUserPerformedAction == true or ActivityInsights.FirstTimeActionPerformedInTenant == true or ActivityInsights.ActionUncommonlyPerformedAmongPeers == true or ActivityInsights.ActionUncommonlyPerformedInTenant == true))"
+      Summarize: summarize ['First Time User Anomaly']=countif(Evidence.FirstTimeUserPerformedAction == true),['Uncommon Among Peers']=countif(Evidence.ActionUncommonlyPerformedAmongPeers == true), ['Uncommon in Tenant']=countif(Evidence.ActionUncommonlyPerformedInTenant == true) , ['First Time Tenant Anomaly']=countif(Evidence.FirstTimeActionPerformedInTenant == True)
+      Project: project Anomaly = 'Anomalous administrative action', ['First Time User Anomaly'], ['Uncommon Among Peers'], ['Uncommon in Tenant'], ['First Time Tenant Anomaly']
+      LinkColumnsDefinitions:
+      - ProjectedName: First Time User Anomaly
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.FirstTimeUserPerformedAction == true)"
+      - ProjectedName: Uncommon Among Peers
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.ActionUncommonlyPerformedAmongPeers == true)"
+      - ProjectedName: Uncommon in Tenant
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.ActionUncommonlyPerformedInTenant == true)"
+      - ProjectedName: First Time Tenant Anomaly
+        Query: "{{BaseQuery}} | {{RowFilter}} and (Evidence.FirstTimeActionPerformedInTenant == True)"
+    - Filter: " where (ActionType in('Sign-in') and  (ActivityInsights.FirstTimeUserConnectedFromCountry == True or  ActivityInsights.FirstTimeConnectionFromCountryObservedInTenant == True or ActivityInsights.CountryUncommonlyConnectedFromInTenant == True or ActivityInsights.CountryUncommonlyConnectedFromAmongPeers == True))"
+      Summarize: summarize ['First Time User Anomaly']=countif(Evidence.FirstTimeUserConnectedFromCountry == True),['Uncommon Among Peers']=countif(Evidence.CountryUncommonlyConnectedFromAmongPeers == True), ['Uncommon in Tenant']=countif(Evidence.CountryUncommonlyConnectedFromInTenant == True) , ['First Time Tenant Anomaly']=countif(Evidence.FirstTimeConnectionFromCountryObservedInTenant == True)
```

Comment:
Need to add 'by UPN' at the end, otherwise when we don't have a match, we will get rows with 0 returns which is not how we want no results to show up in the UX.